includes/class-sensei-question.php

```diff
@@ -641,17 +641,34 @@ public static function get_the_question_media( $question_id ) {
 
 	} // end get_the_question_media
 
-
 	/**
 	 * Output the question media
 	 *
 	 * @since 1.9.0
 	 * @param string $question_id
 	 */
 	public static function the_question_media( $question_id ) {
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo self::question_media_kses_post( self::get_the_question_media( $question_id ) );
+	}
 
-		echo wp_kses_post( self::get_the_question_media( $question_id ) );
+	/**
+	 * Special kses processing for media output to allow 'source' video tag.
+	 *
+	 * @since 3.0.0
+	 * @param string $source_string
+	 * @return string with allowed html elements
+	 */
+	private static function question_media_kses_post( $source_string ) {
+		$source_tag = array(
+			'source' => array(
+				'type' => true,
```

Comment:
I believe that both the `true` values here should be `array()`. 🤔

Reply:
`array()` does also work, but I was following the format defined here: https://core.trac.wordpress.org/browser/tags/5.3/src/wp-includes/kses.php#L61

Reply:
Good to know. I think I like `true` better. 🙂